dear_petition/petition/migrations/0025_auto_20200520_0120.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.12 on 2020-05-20 01:20
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
```

Comment:
We just need to be mindful that we don't have migration conflicts between this PR and DP-35